applications/application_life_cycle_management/odc-working-with-helm-charts-using-developer-perspective.adoc

```diff
@@ -8,6 +8,17 @@ toc::[]
 include::modules/helm-understanding-helm.adoc[leveloffset=+1]
 
 
-You can use the xref:../../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective] in the xref:../../web_console/web-console.adoc#web-console[web console] to create Helm releases.
+You can use the *Developer* perspective in the web console to install a chart from the Helm charts in the developer catalog; create a Helm release; and upgrade, rollback, and uninstall the release.
```

Comment:
s/install a chart from the Helm charts/install a Helm chart?

not sure how accurate is this statement.

Reply:
Reworded to make it more clear.